tests/tensorflow/r2.12/common.libsonnet

```diff
@@ -46,7 +99,7 @@ local mixins = import 'templates/mixins.libsonnet';
       runnerPath: 'official/nlp/train.py',
     },
   },
-  local functional_schedule = '0 3 * * *',
+  local functional_schedule = '0 7 * * *',
```

Comment:
I changed the tests to avoid conflict with nightly tests. Please revert this.

Reply:
I reverted and repushed without these changes